doc/README.md

```diff
@@ -11,6 +10,8 @@ respectively. Some documentation is also stored in the library files to be kept
 close to the code it describes. It is included in the manual using the mechanism 
 documented [here](https://www.gap-system.org/Manuals/pkg/GAPDoc-1.5.1/doc/chap4.html).
```

Comment:
Independent of this PR: we should update this link once GAP 4.11 is out to point at the new version (and/or switch it to http://www.math.rwth-aachen.de/~Frank.Luebeck/GAPDoc/doc/chap4.html )